### DISCUSSION TITLE
Show tracks from hub connection set in the config file

### DISCUSSION THREAD

**Discussion by garethgillard** [GMOD/jbrowse-components#2755] [Q&A]
I have added a hub connection to the jbrowse2-web config with the CLI like so:
`jbrowse add-connection HUB_URL --assemblyName ASSEMBLY --name NAME --type UCSCTrackHubConnection`

This hub I can manually connect to with the 'add connection' button through the web interface and then see all the available tracks. However I don't see the tracks available after adding the connection to the config.json through the CLI.

Can the tracks from a hub connection in the config file be shown under available tracks, and if so how is this setup?

**Comment by cmdcolin** [comment]
Currently the "connections" are sort of hidden behind the "plug" icon in the track selector. You can turn on and turn off connections with this interactively. This might not be the best user experience though, we definitely would consider suggestions to improve

**Reply by garethgillard** [reply]
I see! Can connections be set to on by default through some config setting?

**Reply by garrettjstevens** [reply]
One way to do this would be by setting a default session with that connection open. To see what this might look like, you can open a connection in the UI and then use `File -> Export session`. The JSON file that is exported can be loaded into your config with the `jbrowse set-default-session` command.
